app/Http/Controllers/AuthController.php

```diff
@@ -92,10 +91,15 @@ public function load(Request $request)
 
         $verifiedSignedRequestData = $this->verifySignedRequest($signedPayload);
         if (!$verifiedSignedRequestData) {
-            $errorMessage = 'The signed request from BigCommerce could not be validated';
+            $errorMessage = 'Error: The signed request from BigCommerce could not be validated';
             return new Response($errorMessage, 400);
         } else {
-            $store = Store::where('domain', $verifiedSignedRequestData['context'])->first();
+            $store = Store::where('context', $verifiedSignedRequestData['context'])->first();
```

Comment:
Are we 100% sure `context` exists in this case? I can not find a check ensuring this in `verifySignedRequest`.

Reply:
Extended `if` statement in line 93.